cmd/signing.go

```diff
@@ -285,3 +261,43 @@ func SignDigest(c *cli.Context) error {
 
 	return nil
 }
+
+func outputData(c *cli.Context, data []byte, overwrite bool) error {
+	if outputFilePath := c.String("output-file"); len(outputFilePath) > 0 {
+		var fileMode os.FileMode
+
+		if !overwrite {
```

Comment:
I have 2 possible changes in mind:
- flip logic to get rid of `!`
- or pass `fileMode` as a parameter. I am not sure if we need `os.Stat` check at all, since if a file was written with `0444` then it won't be allowed to be overwritten (permission denied error). If it was written with `0644` then its a green light. Unless I'm missing something here.

Reply:
I like the second proposal for it's simplicity. I'll update the code.